corehq/motech/dhis2/hendler.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import absolute_import
```

Comment:
It's a small thing, I know, but could you rename this module from "hendler" to "handler".